bundles/org.openhab.binding.luftdateninfo/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,106 @@
+			<parameter name="sensorid" type="integer" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="conditions">
+		<label>Condition Sensor</label>
+		<description>Sensor to measure Temperature and Humidity conditions</description>
+
+		<channels>
+			<channel id="temperature" typeId="temp-channel"/>
+			<channel id="humidity" typeId="hum-channel"/>
+			<channel id="pressure" typeId="pressure-channel"/>
+			<channel id="pressure-sea" typeId="pressure-sea-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="integer" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="noise">
+		<label>Noise Sensor</label>
+		<description>Sensor to measure noise on location</description>
+
+		<channels>
+			<channel id="noise-eq" typeId="noise-eq-channel"/>
+			<channel id="noise-min" typeId="noise-min-channel"/>
+			<channel id="noise-max" typeId="noise-max-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="integer" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="pm25-channel">
+		<item-type>Number</item-type>
+		<label>PM 2.5:Density</label>
+		<description>Particulate Matter 2.5</description>
+	</channel-type>
+	<channel-type id="pm100-channel">
+		<item-type>Number</item-type>
+		<label>PM 10.0:Density</label>
+		<description>Particulate Matter 10.0</description>
+	</channel-type>
+	<channel-type id="temp-channel">
+		<item-type>Number</item-type>
+		<label>Temperature:Temperature</label>
+		<description>Temperature from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="hum-channel">
+		<item-type>Number</item-type>
+		<label>Humidity:Dimensionless</label>
+		<description>Humidity from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="pressure-channel">
+		<item-type>Number</item-type>
+		<label>Atmospheric Pressure:Pressure</label>
+		<description>Atmospheric Pressure from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="pressure-sea-channel">
+		<item-type>Number</item-type>
+		<label>Atmospheric Pressure Sea Level:Pressure</label>
+		<description>Atmospheric Pressure at sea level from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="noise-eq-channel">
+		<item-type>Number</item-type>
+		<label>Noise EQ:Dimensionless</label>
+		<description>Average noise level from the selected Sensor ID</description>
+	</channel-type>
+	<channel-type id="noise-min-channel">
+		<item-type>Number</item-type>
+		<label>Min Noise:Dimensionless</label>
+		<description>Minimum noise level (last 2.5 minutes:Dimensionless) from the selected Sensor ID</description>
```

Comment:
This can be removed. Same for below.
```suggestion
		<description>Minimum noise level (last 2.5 minutes) from the selected Sensor ID</description>
```

Reply:
Corrected all locations